ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +20,7 @@ nginx_sites:
    location2:
     name: "/static"
     alias: "{{ nginx_static_home }}"
+    add_header: "Access-Control-Allow-Origin *;"
```

Comment:
that's correct. we just need the static resource to be able to be loaded from a different origin. namely our CDN URL. don't think this really opens us up to any security thing because as you mentioned this is just static files. here are some cURLs:
<img width="941" alt="screen shot 2016-02-28 at 11 06 05 am" src="https://cloud.githubusercontent.com/assets/918514/13380202/7ad5b81c-de0b-11e5-9646-8286f53e22c8.png">